build.sh

```diff
@@ -1,12 +1,28 @@
 #!/bin/sh
 
-set -x
-# script requires the availability of rockcraft, skopeo, yq and docker in the host system
-# it also requires sudo permissions to run skopeo
+# The script requires:
+# - rockcraft
+# - skopeo with sudo privilege
+# - yq
+# - docker
 
-# export version=$(yq -r '.version' rockcraft.yaml)
+set -e
+
+rockcraft="rockcraft.yaml"
+rockcraft_backup="rockcraft_bak.yaml"
+
+restore() {
+    mv "$rockcraft_backup" "$rockcraft"
+    rm -f "$rockcraft_backup"
+}
+trap 'restore' INT TERM EXIT
+
+cp "$rockcraft" "$rockcraft_backup"
+sed -i 's/^base: .*/base: ubuntu@22.04/' "$rockcraft"
```

Comment:
For future reference can you add a comment here saying why we need to do this (something like `use ubuntu base image because we need a shell to export the env vars from a file`)?